driver/asana_driver.py

```diff
@@ -18,12 +19,32 @@
 
 pp = pprint.PrettyPrinter(indent=4)
 
+"""
+1. research what happens when a task/attachment is deleted: [DONE] 404
+2. implement setup/teardown method for driver, handle auth there: [DONE]
+3. switch to auth2
+4. use pagination hash in milestone [ NO ] offset hash not exposed to client lib
+5. use modified_since [DONE]
+"""
 
 def parse_date(s):
     if not s: return None
+    if isinstance(s, datetime.date): return s
```

Comment:
I apologize in advanced for my pedantry, but can you have the body of all if statements on a second line, even if it's just a single statement?